app/hotels/src/appRegistry/HotelsStandalonePackage.js

```diff
@@ -17,7 +19,18 @@ type Props = {|
   checkout?: string,
 |};
 
+type State = {
+  key: string,
+  isTransitioning: boolean,
+  index: number,
+  routes: mixed[],
+};
+
 export default class HotelsStandalonePackage extends React.Component<Props> {
+  navigationController = (previousState: State, currentState: State) => {
+    GestureController(previousState, currentState, 'KiwiHotels');
```

Comment:
I think the logic should happen here. This method should call either `GestureController.addGestures` or `GestureController.removeGestures`. Gesture Controller should have no knowledge of `navigation` state since we want to use it across different navigators in the future.